src/tests/Info.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { render } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
+import '@testing-library/jest-dom/extend-expect';
+import App from '../App';
+
+
+describe('info', () => {
+  it('navigates to ooloo employers page when ooloo.io link is clicked', () => {
+    const history = createMemoryHistory();
+    const { getAllByRole, getByRole } = render(
+      <Router history={history}>
+        <App />
+      </Router>,
+    );
+
+    const infoSection = getByRole('article');
+    const oolooLink = getAllByRole('link', { name: /ooloo\.io/ })[0];
```

Comment:
Instead of `getAllByRole` you could use `within` like this

```
import { within } from "@testing-library/react";

const oolooLink = within(infoSection).getByRole('link', ...) 
```

Makes it a bit more resilient in case there should be another link named ooloo.io somewhere before